src/elements/content-sidebar/SidebarNav.scss

```diff
@@ -64,17 +64,8 @@
     }
 }
 
-.is-responsive-web {
-    @include breakpoint($small-screen) {
```

Comment:
I moved the small-screen block after the medium-screen block so that styles of smaller screens are applied after the styles of larger screens. this is due to the desktop-first media queries approach we're in